src/components/AccordionForm/AccordionForm.js

```diff
@@ -231,6 +234,7 @@ AccordionForm.propTypes = {
         PropTypes.arrayOf(PropTypes.element),
       ]),
       buttonLabel: PropTypes.string,
+      buttonAttribute: PropTypes.object,
```

Comment:
Can this be pluralized?

Reply:
@zeddotes Yes, it can be, just made the change